tests/runtime/pointers

```diff
@@ -0,0 +1,104 @@
+NAME u8 pointer increment
+RUN bpftrace -e 'BEGIN { @=(int8*) 32; @+=1; exit(); }'
+EXPECT ^@: 33
+TIMEOUT 5
+
+NAME u16 pointer increment
+RUN bpftrace -e 'BEGIN { @=(int16*) 32; @+=1; exit(); }'
+EXPECT ^@: 34
+TIMEOUT 5
+
+NAME u32 pointer increment
+RUN bpftrace -e 'BEGIN { @=(int32*) 32; @+=1; exit(); }'
+EXPECT ^@: 36
+TIMEOUT 5
+
+NAME u64 pointer increment
+RUN bpftrace -e 'BEGIN { @=(int64*) 32; @+=1; exit(); }'
+EXPECT ^@: 40
+TIMEOUT 5
+
+NAME u8 pointer unop post increment
+RUN bpftrace -e 'BEGIN { @=(int8*) 32; @++; exit(); }'
+EXPECT ^@: 33
+TIMEOUT 5
+
+NAME u16 pointer unop post increment
+RUN bpftrace -e 'BEGIN { @=(int16*) 32; @++; exit(); }'
+EXPECT ^@: 34
+TIMEOUT 5
+
+NAME u32 pointer unop post increment
+RUN bpftrace -e 'BEGIN { @=(int32*) 32; @++; exit(); }'
+EXPECT ^@: 36
+TIMEOUT 5
+
+NAME u64 pointer unop post increment
+RUN bpftrace -e 'BEGIN { @=(int64*) 32; @++; exit(); }'
+EXPECT ^@: 40
+TIMEOUT 5
+
+NAME u8 pointer unop pre increment
+RUN bpftrace -e 'BEGIN { @=(int8*) 32; @++; exit(); }'
+EXPECT ^@: 33
+TIMEOUT 5
+
+NAME u16 pointer unop pre increment
+RUN bpftrace -e 'BEGIN { @=(int16*) 32; @++; exit(); }'
+EXPECT ^@: 34
+TIMEOUT 5
+
+NAME u32 pointer unop pre increment
+RUN bpftrace -e 'BEGIN { @=(int32*) 32; @++; exit(); }'
+EXPECT ^@: 36
+TIMEOUT 5
+
+NAME u64 pointer unop pre increment
+RUN bpftrace -e 'BEGIN { @=(int64*) 32; @++; exit(); }'
+EXPECT ^@: 40
+TIMEOUT 5
+
+NAME Pointer decrement 1
+RUN bpftrace -e 'BEGIN { @=(int32*) 32; @-=1; exit(); }'
+EXPECT ^@: 28
+TIMEOUT 5
+
+NAME Pointer decrement
+RUN bpftrace -e 'BEGIN { @=(int32*) 32; @--; exit(); }'
+EXPECT ^@: 28
+TIMEOUT 5
+
+NAME Pointer increment 6
+RUN bpftrace -e 'BEGIN { @=(int32*) 32; @+=6; exit(); }'
+EXPECT ^@: 56
+TIMEOUT 5
+
+NAME Pointer decrement 6
+RUN bpftrace -e 'BEGIN { @=(int32*) 32; @-=6; exit(); }'
+EXPECT ^@: 8
+TIMEOUT 5
+
+NAME Struct pointer math
+RUN bpftrace -e 'struct A {int32_t a; int32_t b; char c[28] }; BEGIN { $a=(struct A*) 1024; printf("%lu - 5 x %lu = %lu\n", $a, sizeof(struct A), $a-5); exit(); }'
+EXPECT 1024 - 5 x 36 = 844
+TIMEOUT 5
+
+NAME u8 pointer increment
+RUN bpftrace -e 'BEGIN { @=(int8*) 32; @+=1; exit(); }'
+EXPECT ^@: 33
+TIMEOUT 5
+
+NAME u16 pointer increment
+RUN bpftrace -e 'BEGIN { @=(int16*) 32; @+=1; exit(); }'
+EXPECT ^@: 34
+TIMEOUT 5
+
+NAME u32 pointer increment
+RUN bpftrace -e 'BEGIN { @=(int32*) 32; @+=1; exit(); }'
+EXPECT ^@: 36
+TIMEOUT 5
+
+NAME u64 pointer increment
+RUN bpftrace -e 'BEGIN { @=(int64*) 32; @+=1; exit(); }'
+EXPECT ^@: 40
+TIMEOUT 5
```

Comment:
Aren't these the same tests as the first 4 in the file?

Reply:
hmm think i wanted to modify them, but can't remember into what. Good catch :)

Reply:
Perhaps `s/increment/decrement`? ;-)